client/my-sites/themes/themes-banner/style.scss

```diff
@@ -68,6 +68,28 @@
 		margin-top: 1em;
 	}
 
+	.themes-banner__close {
+		display: block;
+		position: absolute;
+		background: rgba( white, 0.66 );
+		padding: 1px 0;
+		width: 1.7em;
+		line-height: 0.9;
+		z-index: 4;
+		top: 0;
+		right: 0;
+
+		&:hover {
+			cursor: pointer;
+		}
+
+		svg {
```

Comment:
I think we should rather select this by its (more specific) `.gridicon-cross-small` (or whateverit's called ) `className`. (I think it's what we generally do in Calypso. In this case, future-proofness is another argument: Imagine we'd add an SVG as a visual element to a banner -- it'd be erroneously selected by an `svg` selector.)

Reply:
Right, makes total sense indeed!